src/values/UnDeserializableValue.js

```diff
@@ -15,35 +15,35 @@ var PARENT = dv.DataValue;
  *
  * @constructor
  *
- * @param {string} ofType The data value type the structure should have been unserialized to.
- * @param {Object} unUnserializableStructure Plain object assumingly representing some data value
- *        but the responsible unserializer was not able to unserialize it.
- * @param {Error} unserializeError The error thrown during the attempt to unserialize the given
+ * @param {string} ofType The data value type the structure should have been deserialized to.
+ * @param {Object} unDeserializableStructure Plain object assumingly representing some data value
+ *        but the responsible deserializer was not able to deserialize it.
+ * @param {string} deserializeError The error thrown during the attempt to deserialize the given
  *        structure.
  */
 var SELF = dv.UnDeserializableValue = util.inherit(
 	'DvUnDeserializableValue',
 	PARENT,
-	function( ofType, unUnserializableStructure, unserializeError ) {
-		if( !$.isPlainObject( unUnserializableStructure ) ) {
-			throw new Error( 'The un-unserializable structure has to be a plain object' );
+	function( ofType, unDeserializableStructure, deserializeError ) {
+		if ( typeof ofType !== 'string' ) {
```

Comment:
Our JS style guide is different from PHP and uses `if(` with no space. Personally I don't care much, just wanted to let you know.